CHANGELOG.md

```diff
@@ -0,0 +1 @@
+# Changelog
```

Comment:
Is this changelog shared between the python package and the extension?

Reply:
No, it's just for changes that affect projects/pgai.

Reply:
should we nest it under the pgai directory then? We also need a changelog for the extension, or some way to merge the two

Reply:
The changelog I think will be created in projectts/pgai/CHANGELOG.md